### ISSUE TITLE
Different additional Terraform scripts for local/remote deployment

### ISSUE THREAD

**Issue by v3rm0n** [JetBrains/kotless#60]
I need access to SSM so I added a policy to the Lambda execution role, but the same role is not created when I'm running it locally so the additional configuration fails.
So either a possibility to have different additional configurations for local/remote or a way to detect in Terraform scripts if it is running locally. Maybe with a variable :thinking:

**Comment by v3rm0n** [comment]
Would you accept a PR that would add a possiblity to add custom Terraform variables e.g:
```
kotless {
  extensions {
    terraform {
      variables = hashMapOf("myvariable" to "value")
    }
  }
```
And would add a variable for differentiating between local/remote?

**Comment by TanVD** [comment]
Yep, in general, I am ok with it. Let's take a look at the code :)

**Comment by v3rm0n** [comment]
Opened PR, locals made more sense https://github.com/JetBrains/kotless/pull/62
